packages/tools/generator-pie-component/src/app/templates/__package__.json

```diff
@@ -17,7 +17,7 @@
     "test:ci": "yarn test",
     "test:browsers": "npx playwright test -c ./playwright-lit.config.ts",
     "test:browsers:ci": "yarn test:browsers",
-    "test:visual": "run -T cross-env-shell PERCY_TOKEN=${PERCY_TOKEN_PIE_<%= percyComponentName %>} percy exec --allowed-hostname cloudfront.net -- npx playwright test -c ./playwright-lit-visual.config.ts",
```

Comment:
I personally think this is better to have as a generated output. The readme for the generated component might be a good place to detail how to add visual tests.

Reply:
please check this thread and slack chat about this. 
https://github.com/justeattakeaway/pie/pull/669#discussion_r1280853203

the team believes we should throw an error and link to the steps of creating a Percy project rather than having this.